FSW/main.cpp

```diff
@@ -0,0 +1,44 @@
+/* main.cpp is the main script for the NanoSAM II FSW 
+ * Usage:
+ *  run this script through Teensyduino to power the NS2 payload
+ * 
+ * Additional files needed for compilation:
+ *  memUtil.hpp
+ *  memUtil.cpp
+ *
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+#include "headers/memUtil.hpp"
+
+
+/* - - - - - - Functions - - - - - - */
+
+/* - - - - - - main - - - - - - *
+ * Usage:
+ *  Main function entered on runtime, fill with NS2 functionality
+ * 
+ * Inputs:
+ *  none
+ *  
+ * Outputs:
+ *  None
+ */
+int main()
+{
+    // TODO: Change main loop to account for critical exit conditions
+    while(true) // run main loop forever
+    {
+        // test multiple file structure by calling functions from memUtil.cpp
+        dataProcessing();
+        scienceMemoryHandling();
+        break;
```

Comment:
Right now it breaks upon completion for testing, but how does this loop run in reality? After a critical exit condition occurs, is there methodology to get back into the loop? Something like

```
begin
    function()
rescue
    handleException()
    continue
end
```

Reply:
@Vysares I will defer to you on this one, I just put this main loop in here for super basic testing but I agree with Jashan that this will need to be fleshed out eventually